registry.md

```diff
@@ -1,68 +1,62 @@
+    - [Ethereum Networks](#ethereum-networks)
+- [Assets](#assets)
+  - [`bitcoin` Parameters](#bitcoin-parameters-1)
+  - [`ether` Parameters](#ether-parameters)
+  - [`erc20` Parameters](#erc20-parameters)
+- [Protocols](#protocols)
+- [Negotiation Errors](#negotiation-errors)
+  - [Details of the `timeouts-too-tight` error](#details-of-the-timeouts-too-tight-error)
+- [Identities](#identities)
+- [Hash Functions](#hash-functions)
+- [Headers](#headers)
+
 ## Description
 
-This registry defines all types used across COMIT RFCs.
-This registry may be expanded with new RFCs.
+This registry defines all types used across COMIT RFCs. This registry may be expanded with new RFCs.
 
 ## Ledgers
 
 The following is a list of possible values a `Ledger` type header can take:
 
-| Value      | Reference                            | Description                            |
-|:-----------|--------------------------------------|----------------------------------------|
-| `bitcoin`  | [RFC-004](./RFC-004-Bitcoin.md) | The Bitcoin Core family of blockchains |
-| `ethereum` | [RFC-006](./RFC-006-Ethereum.md)| A blockchain following the Ethereum consensus rules     |
-
+| Value      | Reference                        | Description                                         |
+| :--------- | -------------------------------- | --------------------------------------------------- |
+| `bitcoin`  | [RFC-004](./RFC-004-Bitcoin.md)  | The Bitcoin Core family of blockchains              |
+| `ethereum` | [RFC-006](./RFC-006-Ethereum.md) | A blockchain following the Ethereum consensus rules |
 
 And the possible parameters they each may have:
 
 ### `bitcoin` Parameters
 
-| Parameter | Value Type                                  | Description                              |
-|:----------|---------------------------------------------|------------------------------------------|
+| Parameter | Value Type                                | Description                              |
+| :-------- | ----------------------------------------- | ---------------------------------------- |
 | `network` | see [Bitcoin Networks](#bitcoin-networks) | The particular blockchain network to use |
 
 #### Bitcoin Networks
 
 | Value     | Description                          |
-|:----------|:-------------------------------------|
+| :-------- | :----------------------------------- |
 | `regtest` | Private Bitcoin Core regtest network |
 | `testnet` | Bitcoin Core testnet                 |
 | `mainnet` | Bitcoin Core mainnet                 |
 
-
 ### `ethereum` Parameters
 
-| Parameter | Value Type                                    | Description                              |
-|:----------|-----------------------------------------------|------------------------------------------|
+| Parameter | Value Type                                  | Description                              |
+| :-------- | ------------------------------------------- | ---------------------------------------- |
```

Comment:
It would be great if all this useless auto-reformat were done in another PR to not clutter this PR :)
When `cargo fmt` for markdown?

Reply:
Too late at this stage :see_no_evil: